src/components/ui/ImageGallery/ImageZoom.tsx

```diff
@@ -0,0 +1,13 @@
+import type { PropsWithChildren } from 'react'
+import React from 'react'
+
+interface ImageZoomProps {
+  helpMessage?: string
+  zoomFactor?: number
+}
+
+const ImageZoom = ({ children }: PropsWithChildren<ImageZoomProps>) => {
+  return <> {children} </>
```

Comment:
```suggestion
  return children
```

Reply:
I tried adding your change, but then I got the following error at `ImageGallery.tsx` when using the `<ImageZoom />`:

```
'ImageZoom' cannot be used as a JSX component.
  Its return type 'ReactNode' is not a valid JSX element.
    Type 'undefined' is not assignable to type 'Element | null'
```